docs/src/main/paradox/cleanup.md

```diff
@@ -0,0 +1,55 @@
+# Database Cleanup
+
+## Event Sourced cleanup tool
+
+@@@ warning
+
+When running any operation with the `EventSourcedCleanup`  for a persistence id the actor with that persistence id must
```

Comment:
Maybe either use "with the `EventSourcedCleanup` tool" or just "with `EventSourcedCleanup`"

```suggestion
When running any operation with `EventSourcedCleanup` for a persistence id, the actor with that persistence id must
```